cpu/stm32/periph/rtc_all.c

```diff
@@ -221,7 +221,7 @@ static inline void rtc_unlock(void)
     while (!(RTC_REG_ISR & RTC_ISR_INITF)) {}
 }
 
-static inline void rtc_lock(void)
+extern void rtc_lock(void)
```

Comment:
I still don't get what the `extern` is doing here

Reply:
You say because the `extern` is implicit? Ok will remove.